src/main/java/uk/co/ramp/covid/simulation/population/Person.java

```diff
@@ -84,6 +84,13 @@ public int hashCode() {
         return personId;
     }
     
+    public double setMortality() {
+    	double out = 0.0;
+    	if(age > 50) out = Math.pow((((double) age - 50.0) / 50.0) + CovidParameters.get().diseaseParameters.caseMortalityBase, 2.0);
```

Comment:
Do we want this to be settable in case new data appears?

Reply:
Could do. Let's leave it for now and we can always pull it out